Re-Onboarding does not work after reboot, as "iotedge config apply" tries to connect to iotedge-docker
as it is still found in leftover config.toml and denies working with "File not found". Starting
iotedge-docker before calling "iotedge config apply" fixes this problem
diff -aurN a/pkg/onboard/components/Manual/Manual.js b/pkg/onboard/components/Manual/Manual.js
--- a/pkg/onboard/components/Manual/Manual.js	2023-08-04 07:42:43.000000000 +0200
+++ b/pkg/onboard/components/Manual/Manual.js	2024-02-21 17:50:32.944344747 +0100
@@ -539,6 +539,7 @@
                 proxySettings,
             };
             await this.updateOnboardJSON(onboardData);
+            await misc.spawn(["systemctl", "start", "iotedge-docker"]);
             await misc.spawn(["iotedge", "config", "apply"]);
             await misc.spawn(["systemctl", "enable", "aziot-edged"]);
             this.displayStatusMessage(
@@ -835,6 +836,7 @@
                                             proxySettings
                                         };
                                         await this.updateOnboardJSON(onboardData);
+                                        await misc.spawn(["systemctl", "start", "iotedge-docker"]);
                                         await misc.spawn(["iotedge", "config", "apply"]);
                                         await misc.spawn(["systemctl", "enable", "aziot-edged"]);
                                         this.displayStatusMessage(
@@ -1084,6 +1086,11 @@
                                                 onboardData
                                             );
                                             await misc.spawn([
+                                                "systemctl",
+                                                "start",
+                                                "iotedge-docker"
+                                            ]);
+                                            await misc.spawn([
                                                 "iotedge",
                                                 "config",
                                                 "apply",
